contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -3221,10 +3221,170 @@ where has_schema_privilege(sch.schema_id, 'USAGE')
 and has_table_privilege(c.oid, 'SELECT,INSERT,UPDATE,DELETE,TRUNCATE,TRIGGER');
 GRANT SELECT ON sys.index_columns TO PUBLIC;
 
+
+ALTER FUNCTION sys.json_modify RENAME TO json_modify_deprecated_in_3_1_0;
```

Comment:
please add minor version upgrade tests which validate dependency of user views on this function. Adding to latest schedule is not enough as it does not test for minor version upgrade. Test should trigger 3.0 to 3.1 upgrade script.

Reply:
This was supposed to be in 3.1-3.2 so I just moved it.  The json_modify function was originally implemented in 14_6 and the file testing it (babel-937) is in the upgrade schedule for 14_6, 14_7, 14_8, 15_1, 15_2, and 15_latest. I modified the function and deprecated it in the 3.1-3.2 upgrade script, and added my test file to the 15_latest schedule file. I double checked with Jungkook and this should be sufficient for ensuring customers will not face any errors using json_modify in older versions